packages/react-ui-core/src/Photo/Photo.js

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(['Photo'], {
+  pure: true,
+})
+
+export default class Photo extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string,
+    fallbackUrl: PropTypes.string,
+    theme: PropTypes.object,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  static getDerivedStateFromProps(props, state) {
+    if (!state.url || (state.url !== props.url)) {
+      return {
+        url: props.url,
+        error: false,
+      }
+    }
+
+    return null
+  }
+
+  fallback() {
+    this.setState({ error: true })
+  }
+
+  render() {
+    const {
+      theme,
+      className,
+      url: _,
+      fallbackUrl,
+      ...rest
+    } = this.props
+
+    const { url, error } = this.state
+
+    /* eslint-disable jsx-a11y/alt-text */
```

Comment:
I assume you can pass `alt` as one of the "rest" params. Since that is a lint rule in the first place, what is our policy of always specifying an `alt` attribute? Should it be specified a prop and encourage to be used?

Reply:
Yeah, I was thinking maybe we just want to enforce it. 🤷‍♂️